src/js/jsx/search/SearchBar.jsx

```diff
@@ -308,6 +360,7 @@ define(function (require, exports, module) {
                     if (this.refs.datalist.cursorAtBeginning() && this.state.filter.length > 0) {
                         // Clear filter and icons
                         this._updateFilter(null);
+                        this.refs.datalist.forceUpdate();
```

Comment:
Bad indentation below in `render`. See other JSX files to see how we indent multi-line component expressions.